src/index.js

```diff
@@ -1,14 +1,15 @@
+/* eslint-disable no-restricted-globals, prefer-destructuring, import/no-unresolved */
 import './style.css';
 import todoStatusUpdate from './todoStatusUpdate.js';
 import dragDrop from './dragDrop.js';
+import deleteCompletedTodoItem from './deleteAll.js';
+import addTodo from './addTodo.js';
+import edit from './editTask.js';
```

Comment:
- [ ] There is this error ⬇️ due to a typo , its looking for a file named `./addTodo.js` but this is what exists `/addToDo.js`. Kindly fix this
![Screenshot from 2021-08-20 11-34-13](https://user-images.githubusercontent.com/33067256/130205290-7603644b-f523-42b2-9253-a824f7620a2a.png)